extensions/ql-vscode/src/common/commands.ts

```diff
@@ -12,6 +12,7 @@ import type {
   VariantAnalysisScannedRepositoryResult,
 } from "../variant-analysis/shared/variant-analysis";
 import type { QLDebugConfiguration } from "../debugger/debug-configuration";
+import { QueryTreeViewItem } from "../queries-panel/query-tree-view-item";
```

Comment:
This should be a `type` import to ensure this file remains somewhat independent of `vscode` and only needs the types from `vscode`, rather than the complete `vscode` module:

```suggestion
import type { QueryTreeViewItem } from "../queries-panel/query-tree-view-item";
```